devices/Adxl343/ADXL343Samples/Program.cs

```diff
@@ -0,0 +1,59 @@
+//
+// Copyright (c) 2017 The nanoFramework project contributors
+// See LICENSE file in the project root for full license information.
+//
+
+using System;
+using System.Device.I2c;
+using System.Diagnostics;
+using System.Numerics;
+using System.Threading;
+using nanoFramework.Hardware.Esp32;
+using Iot.Device.ADXL343;
+using Iot.Device.ADXL343Lib;
+
+namespace ADXL343Samples
+{
+    public class Program
+    {
+        private const bool useESP32 = true;
+        private const int ESP32DataPin = 22;
+        private const int ESP32ClockPin = 20;
+        private const int i2cBusId = 1;
+        private const int i2cAddr = 0x53;
+
+        private static I2cDevice i2c;
+        public static void Main()
+        {
+            Debug.WriteLine("Hello from ADXL343!");
+
+            //////////////////////////////////////////////////////////////////////
+            // when not connecting to an ESP32 device, there is not need to configure the GPIOs
+            // used for the bus
+            if (useESP32)
```

Comment:
rather than this "if" condition, use a #if define one because you'll need to get rid of the associated using when the sample is used in another board